aten/src/TH/THTensor.h

```diff
@@ -21,6 +21,9 @@
 #include <TH/generic/THTensorRandom.h>
 #include <TH/THGenerateAllTypes.h>
 
+#include <TH/generic/THTensorRandom.h>
+#include <TH/THGenerateBoolType.h>
```

Comment:
this generates code for random, clampedRandom, and cappedRandom -- do we have tests for all of these with bool?